packages/swarm-docs/src/assets/docs.css

```diff
@@ -2,14 +2,127 @@
   margin: 10px 0;
 }
 
+h1.docs-title {
+  margin: 60px 0 40px;
+}
+
+h1.docs-title a {
+  font-weight: var(--font-weight-normal);
+  color: var(--color-gray-5);
+}
+
+.main {
+  margin: 0 0 90px 320px;
+  width: 810px;
+}
+
+
+
+/* Left Navigation */
 .left-nav {
   position: fixed;
   top: 0;
   left: 0;
-  width: 90px;
-  padding: 10px;
+  bottom: 0;
+  width: 200px;
+  padding: 40px 30px 30px;
+  background: var(--color-gray-1);
+}
+.left-nav ul,
+.left-nav li {
+  list-style: none;
+  margin: 0;
+  padding: 0;
 }
 
-.main {
-  margin-left: 100px; 
+.left-nav a {
+  color: var(--color-gray-7);
 }
+
+.left-nav a:hover {
+  color: var(--color-gray-6);
+}
+.swarm-logo {
+  display: block;
+  margin: 0 auto var(--size-space-double);
+}
+
+
+/* Props Table */
+.props-table {
+  width: 100%;
+  border-collapse: separate;
+  border-spacing: 5px;
+  margin-left: -5px;
+}
+
+
+
+.props-table thead,
+.props-table tbody {
+  width: 100%;
+}
+
+
+.props-table th{
+  border-bottom: 2px solid var(--color-indigo);
+  margin-right: 5px;
+  color: var(--color-gray-7);
+}
+
+.props-table tfoot th{
+  border-bottom: 0;
+  border-top: 2px solid var(--color-indigo);
+}
+
+.props-table th,
+.props-table td {
+  padding: 10px 20px 10px 0;
+}
+
+.props-table td,
+.props-table td span {
+  color: var(--color-gray-5);
+  font-size: var(--font-size-small);
+}
+
+.props-table tr:hover {
+  /*background-color: var(--color-gray-1);*/
```

Comment:
Should this stay commented out or do you want to just remove?

Reply:
I'll remove it, thanks